src/StringParser/Json.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Noodlehaus\StringParser;
+
+use Noodlehaus\Exception\ParseException;
+
+/**
+ * JSON string parser
+ *
+ * @package    Config
+ * @author     Jesus A. Domingo <jesus.domingo@gmail.com>
```

Comment:
Please, add yourself as author of this class, removing the others.

Reply:
I've fixed this.